backend/src/main/kotlin/com/tngtech/apicenter/backend/connector/database/entity/SpecificationEntity.kt

```diff
@@ -18,14 +18,14 @@ class SpecificationEntity(
     val remoteAddress: String?
 ) {
 
-    fun pureUpdate(other: SpecificationEntity): SpecificationEntity {
-        val versions = this.versions + other.versions
+    fun appendVersion(other: VersionEntity): SpecificationEntity {
+        val versions = this.versions + other
         return SpecificationEntity(
                 this.id,
                 other.title,
```

Comment:
I find it odd that we use sometimes `this` and sometimes `other`. Can we make it as consistent as possible, so it's easy to see where the differences are?

Reply:
In fact, I was considering removing title and description from the Specification object. As you see here, these fields are sourced / copied directly from the latest version. So really, this method is always retaining the important details from `this`, ie. the pre-existing specification the new version is to be appended to.

Reply:
Where do suggest we store title and description?

Reply:
What I mean is that title / description in the top-level SpecificationEntity object is redundant, if these fields are directly sourced from the latest version contained within the object. I couldn't see a reason for this duplication.